From b217d2dad5bec3311acb409e8e8218fbe4bdd142 Mon Sep 17 00:00:00 2001
From: c_shengl <shengl@codeaurora.org>
Date: Tue, 19 Jan 2016 15:09:03 +0800
Subject: [PATCH 02/12] Add multi reminders

  New a event,add reminders,cannot add multi reminders.

  Change the max reminders to 5.

  CRs-fixed: 955855

Change-Id: I83e391b94088659506ce0afda94965f103da308a
---
 src/com/android/exchange/utility/CalendarUtilities.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/com/android/exchange/utility/CalendarUtilities.java b/src/com/android/exchange/utility/CalendarUtilities.java
index a37b1020..4609bf55 100644
--- a/src/com/android/exchange/utility/CalendarUtilities.java
+++ b/src/com/android/exchange/utility/CalendarUtilities.java
@@ -1433,7 +1433,7 @@ public class CalendarUtilities {
         // Don't show attendee status if we're the organizer
         cv.put(Calendars.CAN_ORGANIZER_RESPOND, 0);
         cv.put(Calendars.CAN_MODIFY_TIME_ZONE, 0);
-        cv.put(Calendars.MAX_REMINDERS, 1);
+        cv.put(Calendars.MAX_REMINDERS, 5);
         cv.put(Calendars.ALLOWED_REMINDERS, ALLOWED_REMINDER_TYPES);
         cv.put(Calendars.ALLOWED_ATTENDEE_TYPES, ALLOWED_ATTENDEE_TYPES);
         cv.put(Calendars.ALLOWED_AVAILABILITY, ALLOWED_AVAILABILITIES);
-- 
2.17.1

